cmd/esctl/main.go

```diff
@@ -49,9 +51,16 @@ func setup() {
 		fmt.Println(err)
 		os.Exit(1)
 	}
+	if len(os.Getenv("ESCTL_GEN_DOCS")) > 0 {
+		err = doc.GenMarkdownTree(rootCmd, "./docs")
+		if err != nil {
+			log.Fatal(err)
```

Comment:
Add context in logs for easier debugging

Reply:
yeah missed it here adding.